### DISCUSSION TITLE
Volume Mixer

### DISCUSSION THREAD

**Discussion by EaZyStudiio** [File-New-Project/EarTrumpet#1126] [Ideas]
![NVIDIA_Share_cGkXM16dAs](https://user-images.githubusercontent.com/55167211/164433577-06d6ce4a-a7a2-428a-86d8-67732dfdec05.png)

I am not quite sure if this is the direction EarTrumpet wants to go BUT this is definitely a great addition.

The screenshot above takes almost 10 clicks to make the changes it would be great if ear-trumpet could make it in 3.

I think the image explains enough but if it does not read on.

The feature above allows you to Change either your input or output device, to any connected device you have on your windows machine  (WM) For Example u can change from ur WM Speakers to your headset or vise-versa

**Comment by riverar** [comment]
Hi @EAZY5D, we have this feature! Right-click any app in EarTrumpet and click the double-arrow icon. (Screenshot below.)

![image](https://user-images.githubusercontent.com/475132/164507634-89b3e39e-7454-449b-b273-6264bbaea616.png)

**Reply by EaZyStudiio** [reply]
Wow that's great thank u
